packages/lesmis/src/components/MapOverlaysPanel/MapOverlaysPanel.js

```diff
@@ -32,6 +32,29 @@ const Box = styled(MuiBox)`
   margin-left: 1rem;
 `;
 
+const searchOverlays = (result, child, path, selectedOverlay) => {
```

Comment:
What was it that we worked on together that was super similar to this?

Reply:
It was processing overlays to only show the visible overlays for each entity level. It's the same nested data structure which is why the recursive functions are quite similar.